R/auth.R

```diff
@@ -64,8 +48,21 @@ get_api_key <- function() {
 }
 
 #' @rdname get_api_key
-#' @param key API key to use for future requests
 #' @export
-set_api_key <- function(key) {
-  options(delphi.epidata.key = key)
+save_api_key <- function() {
+  cli::cli_inform(
+    c(
+      "i" = "This function will open your .Renviron file in a text editor. You will need to
+      write DELPHI_EPIDATA_KEY=yourkeyhere in the file and save it. If the editor
+      does not open, you will need to edit the file manually.",
+      "i" = "Press any key to continue."
```

Comment:
```suggestion
      "i" = "This function will open your {.code .Renviron} file in a text editor. You will need to
      write {.code DELPHI_EPIDATA_KEY=yourkeyhere} in the file and save it. If the editor
      does not open, you will need to edit the file manually.",
      "i" = "Press any key to continue."
```

for formatting in rich-text terminals, like RStudio